sbi/inference/sre/sre.py

```diff
@@ -197,15 +198,16 @@ def _train(
     ):
         """
         Trains the classifier by maximizing a Bernoulli likelihood which distinguishes
-        between jointly distributed (parameter, observation) pairs and randomly chosen
-        (parameter, observation) pairs.
+        between jointly distributed (theta, x) pairs and randomly chosen (theta, x)
+        pairs.
+
         Uses early stopping on a held-out validation set as a terminating condition.
         """
 
         # Get total number of training examples.
-        num_examples = torch.cat(self._parameter_bank).shape[0]
+        num_examples = torch.cat(self._theta_bank).shape[0]
```

Comment:
We're really concatenating here just to get a shape, that's crazy! (but again, offtopic here).